lib/ex_unit/lib/ex_unit/on_exit_handler.ex

```diff
@@ -1,39 +1,75 @@
 defmodule ExUnit.OnExitHandler do
   @moduledoc false
+
   @name __MODULE__
+  @supervisor 2
+  @on_exit 3
```

Comment:
A record would work too, that's what I would use if we add a new element to the tuple.